cmd/shortener/main.go

```diff
@@ -56,7 +56,7 @@ func run(ctx context.Context) error {
 		return err
 	}
 
-	storage, err := getStorage(ctx, cfg)
+	storage, finderStats, err := getStorage(ctx, cfg)
```

Comment:
Я бы предложил `statsFinder`, хотя это и не принципиально. 
```suggestion
	storage, statsFinder, err := getStorage(ctx, cfg)
```